Makefile

```diff
@@ -223,6 +223,10 @@ test-gitlab2:
 lint:
 	$(MAKE) -C spyglass lint_rtl
 
+# IPStools Integration
+ipstools:
+	pip install -t ipstools --upgrade git+https://github.com/pulp-platform/IPApprox.git@master
```

Comment:
I think we can use this change to not always install `master`, but some fixed version which we progressively update. That means we should also make release tags for `ipapprox`.